Assets/MixedRealityToolkit/Utilities/Scripts/Editor/CustomHeaderDrawer.cs

```diff
@@ -1,15 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Reflection;
-using System.Text.RegularExpressions;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
 using UnityEngine;
-#if UNITY_EDITOR
+using MixedRealityToolkit.UtiltiesInspectors;
```

Comment:
typo: UtiltiesInspectors -> UtilitiesInspectors

Reply:
Good catch, darn Copy and Paste.
it's in a few files and darn it I missed it.  Global fix incoming

Reply:
Sounds good!